llvm/include/llvm/ObjCopy/CommonConfig.h

```diff
@@ -245,6 +253,9 @@ struct CommonConfig {
   StringMap<uint64_t> SetSectionType;
   StringMap<StringRef> SymbolsToRename;
 
+  // Adjust section VMA and LMA
```

Comment:
```suggestion
  // Adjust section VMA and LMA.
```
Comments should use correct punctuation. That being said, this comment doesn't add any value and the member should just be added to the immediately above set of `StringMap` members.